src/ext/java/org/opentripplanner/ext/legacygraphqlapi/LegacyGraphQLUtils.java

```diff
@@ -135,4 +136,11 @@ public static boolean startsWith(String str, String name, Locale locale) {
   public static boolean startsWith(I18NString str, String name, Locale locale) {
     return str != null && str.toString(locale).toLowerCase(locale).startsWith(name);
   }
+
+  /**
+   * Converts iterable to List or returns an empty List of iterable is null.
+   */
+  public static <T> List<T> mapIterableToList(Iterable<T> iterable) {
+    return iterable == null ? List.of() : (List<T>) iterable;
```

Comment:
Even though it's very likely that the `Iterable` is actually a `List` in a library function I would probably do a proper conversion to avoid `ClassCastingExceptions`. You could choose something from https://www.baeldung.com/java-iterable-to-collection

Reply:
```suggestion
    return iterable == null ? List.of() : java.util.stream.StreamSupport.stream(iterable.spliterator(), false).toList();
```

There you go. I think it's easier to do than to debate it. :smile:

Reply:
Thanks, it is sometimes... :D